core-services/image-mirroring/openshift/mapping_origin_4_14

```diff
@@ -167,6 +167,7 @@ registry.ci.openshift.org/origin/4.14:ingress-node-firewall-daemon quay.io/opens
 registry.ci.openshift.org/origin/4.14:insights-operator quay.io/openshift/origin-insights-operator:4.14 quay.io/openshift/origin-insights-operator:4.14.0 quay.io/openshift/origin-insights-operator:latest
 registry.ci.openshift.org/origin/4.14:installer quay.io/openshift/origin-installer:4.14 quay.io/openshift/origin-installer:4.14.0 quay.io/openshift/origin-installer:latest
 registry.ci.openshift.org/origin/4.14:installer-artifacts quay.io/openshift/origin-installer-artifacts:4.14 quay.io/openshift/origin-installer-artifacts:4.14.0 quay.io/openshift/origin-installer-artifacts:latest
+registry.ci.openshift.org/origin/4.14:installer-terraform-artifacts quay.io/openshift/origin-installer-terraform-artifacts:4.14 quay.io/openshift/origin-installer-terraform-artifacts:4.14.0 quay.io/openshift/origin-installer-terraform-artifacts:latest
```

Comment:
hm. how/why was this generated?

Reply:
By promoting the image to the ocp namespace in the config.

Reply:
And by running `make openshift-image-mirror-mappings`

Reply:
gh is missing the button to `Resolve Conversation`? This is clear now.